src/set.rs

```diff
@@ -149,14 +135,7 @@ where
         Q: Hash + Eq,
         F: FnOnce(&Q) -> T,
     {
-        match self.map.get_key_value(value) {
-            Some((k, _v)) => k,
-            None => {
-                let (bucket, _) = unsafe { self.map.raw_insert(f(value), (), init) };
-
-                unsafe { &bucket.as_ref().0 }
-            }
-        }
```

Comment:
Please revert unrelated changes

Reply:
It is related because this PR is `Initialize CI`.
You need to know these implementations occurs compile error.
It is not advisable to push code with compilation errors, even if there is no CI.

But it has already been pushed.
So it is necessary to eliminate any code that could disrupt the CI process.